staging/vhost-device-gpu/src/protocol.rs

```diff
@@ -17,76 +17,64 @@ use std::{
 
 use rutabaga_gfx::RutabagaError;
 use thiserror::Error;
+use virtio_bindings::virtio_gpu::{
```

Comment:
You can put `pub use virtio_bindings::virtio_gpu::{` here `pub use` will reexport the symbols under the new name. And then in  `device.rs` I think we can just do `use protocol::*`.